tensorflow_addons/optimizers/lamb_test.py

```diff
@@ -397,21 +341,58 @@ def loss():
         # Validate updated params
         self.assertAllClose(self.evaluate(w), [0.4, 0.2, -0.5], rtol=1e-2, atol=1e-2)
 
-    def test_get_config(self):
-        opt = lamb.LAMB(1e-4)
-        config = opt.get_config()
-        self.assertEqual(config["learning_rate"], 1e-4)
-
-    def test_exclude_weight_decay(self):
-        opt = lamb.LAMB(
-            0.01, weight_decay_rate=0.01, exclude_from_weight_decay=["var1"]
-        )
-        assert opt._do_use_weight_decay("var0")
-        assert not opt._do_use_weight_decay("var1")
-        assert not opt._do_use_weight_decay("var1_weight")
-
-    def test_exclude_layer_adaptation(self):
-        opt = lamb.LAMB(0.01, exclude_from_layer_adaptation=["var1"])
-        assert opt._do_layer_adaptation("var0")
-        assert not opt._do_layer_adaptation("var1")
-        assert not opt._do_layer_adaptation("var1_weight")
+
+@pytest.mark.usefixtures("maybe_run_functions_eagerly")
+def do_test_basic():
```

Comment:
The function is not recognized as a test if the name is not `test_...`. See https://docs.pytest.org/en/latest/goodpractices.html#conventions-for-python-test-discovery .